Dockerfile

```diff
@@ -35,7 +36,7 @@ COPY $WEBHOOK_DIR/src ./$WEBHOOK_DIR/src
 
 COPY tools ./tools
 
-RUN --mount=type=cache,target=/root/.m2 mvn -ntp clean install
+RUN --mount=type=cache,target=/root/.m2 mvn -ntp clean install -DskipTests=$SKIP_TESTS
```

Comment:
I have also think about this, can we just mvn build when building docker image, Because the CI already run the test. (Make the `Build with Maven` not skipTests as now)

Reply:
Get it.